.helm/templates/crd-rest-api-fa.yaml

```
@@ -61,7 +61,7 @@ spec:
                     apiGroup: streaming.sneaksanddata.com
                     kind: StreamingJobTemplate
                     name: standard-job
-                reloadingJobTemplateRef:
+                backfillJobTemplateRef:
                   description: |
                     Name of the job template to be used for the streaming job if stream is running in reloading mode.
```

Should we also update the descriptions for consistency?
```suggestion
                    Name of the job template to be used for the streaming job if stream is running in backfill mode.
```